### DISCUSSION TITLE
Upload file Cannot mkdir in a snapshot. Try mountpoints instead.

### DISCUSSION THREAD

**Discussion by Denise-David** [feathersjs/feathers#2458] [Q&A]
Good morning,
I created an app with the upload of feathers.js, locally everything works perfectly but when I publish the whole upload it gives me this problem:
Cannot mkdir in a snapshot. Try mountpoints instead.

What does it mean?

**Comment by daffl** [comment]
If you are deploying a container it needs to have a writable folder it can access. You can find this information in the documentation of your deployment platform (e.g. [here is an article for Docker](https://towardsdatascience.com/how-to-mount-a-directory-inside-a-docker-container-4cee379c298b)).
